### PR TITLE
Create proselint.yml

### PR DESCRIPTION
**Pull request by BDadmehr0**
- **Specifying Files**: Modify `proselint **/*.md` to include specific file types or directories as needed.
- **Integration with PRs**: This setup will make it easy to catch prose issues in pull requests before merging.
- This `.proselintrc` file can enable or disable specific checks. Refer to the [Proselint documentation](https://github.com/amperser/proselint) for a full list of options.

### CODE REVIEW

**Comment by BaseMax** [comment]
Many thanks. @jbampton please check it in the morning.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -20,8 +20,8 @@ repos:
       - id: prettier
         name: run prettier
         description: format files with prettier
-        entry: prettier --write .
-        files: \.(css|html|md|ya?ml)$
+        entry: prettier --write . # Check formatting without modifying files
+        filies: \.(css|html|md|ya?ml)$
```
Comment (jbampton) [inline .pre-commit-config.yaml:24]:
```suggestion
        files: \.(css|html|md|ya?ml)$
```

**Review by jbampton — APPROVED** [review]
🚀

**Comment by jbampton** [comment]
Thank you !!

**Pull request merged**